spec/std/system_spec.cr

```diff
@@ -4,8 +4,10 @@ require "system"
 describe System do
   describe "hostname" do
     pending_win32 "returns current hostname" do
+      pending! "The `hostname` command is not available." if Process.find_executable("hostname").nil?
+
       shell_hostname = `hostname`.strip
-      $?.success?.should be_true # The hostname command has to be available
+      $?.success?.should be_true
```

Comment:
I don't think we need to check existence beforehand. We can just skip the spec if process status indicates failure, which includes failures in `hostname` itself as well.

```suggestion
      pending! "`hostname` command was unsuccessful" unless $?.success?
```